lcov.coverage

```diff
@@ -0,0 +1,1469 @@
+TN:
```

Comment:
idk if we can put in on gitignore ? I don't see anything on doc ...

Reply:
I think it couldn't be gitignored because i think this is what the badge checks against.

But i'm not sure if this even works: I dunno how to get the badge to consume it.